ui/imports/shared/stores/PermissionsStore.qml

```diff
@@ -97,13 +97,13 @@ QtObject {
         id: becomeMemberPermissionsModel
         sourceModel: root.permissionsModel
         function filterPredicate(modelData) {
-            return (modelData.permissionType == Constants.permissionType.member) &&
-                (modelData.tokenCriteriaMet || !modelData.isPrivate)
+            return (modelData.permissionType === Constants.permissionType.member ||
+                    modelData.permissionType === Constants.permissionType.becomeTokenMaster)
```

Comment:
1. If we filter permissions that are private here, we will not be able to display them in case of met, since they will not be part of the model. That's why in the store we need to keep all permissions related to joining a commuity and the corresponding view, in this case, `overlayPanel` will be in charge of filtering according to the UI needs. 
And UI needs are:
- If permissions are private / hidden, BUT the user mets them, they are shown as part of the list of permissions.
- If permissions are private / hidden, BUT the user does NOT met them, they are NOT shown as part of the list of permissions.
3. This model is a  `join permision members model` so, I don't understand why we need the `admin` permission here.